CMakeLists.txt

```diff
@@ -407,6 +407,10 @@ endif()
 
 find_program(LUACHECK_PRG luacheck)
 
+if(NOT GPERF_PRG)
```

Comment:
Can be just:

```
find_program(GPERF_PRG gperf)
```

If the variable is already set,  it won't be overwritten by find_program.